### PR TITLE
docs: fix some typo and bad links in getting-started.md

### PR DESCRIPTION
**Pull request by suyanhanx**
Signed-off-by: han <suyanhanx@gmail.com>

### CODE REVIEW

File: docs-zh/guide/getting-started.md [hunk 1]
```diff
@@ -217,18 +217,18 @@ envd-quick-start        http://localhost:42779   envd-quick-start.envd   /home/g
 
 ## 更多文档 📝
 
-访问[envd 文档](https://envd.tensorchord.ai/guide/getting-started.html)。
+访问 [envd 文档](../guide/getting-started)。
```
Comment (kemingy) [inline docs-zh/guide/getting-started.md:221]: Linked to itself. Maybe we can delete this part.

File: docs/guide/getting-started.md [hunk 2]
```diff
@@ -209,18 +209,18 @@ envd-quick-start        http://localhost:42779   envd-quick-start.envd   /home/g
 
 ## More on documentation 📝
 
-See [envd documentation](https://envd.tensorchord.ai/guide/getting-started.html).
+See [envd documentation](../guide/getting-started).
```
Comment (kemingy) [inline docs/guide/getting-started.md:213]: The same here.

File: docs-zh/guide/getting-started.md (hunk 1 above)
Comment (suyanhanx) [reply docs-zh/guide/getting-started.md]: Done

File: docs/guide/getting-started.md (hunk 2 above)
Comment (suyanhanx) [reply docs/guide/getting-started.md]: Done.

**Review by kemingy — APPROVED** [review]

**Pull request merged**